### PR TITLE
#4049 Add tests to verify delete workspace works normally

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4049)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/e2e/deleteWorkspace.test.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    cy.findByText('workspace1').click();
+    cy.contains('li', 'Settings').click();
+
+    // click the delete workspace button
+    cy.findByText('Delete this workspace').click();
+    cy.findByPlaceholderText('workspace1').type('workspace1');
+
+    cy.findByText('Delete').click();
+
+    // wait a seconds
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/deleteWorkspace.test.ts:59]: a _few_ seconds :)

File: ohara-manager/client/src/store/epics/worker/fetchWorkerEpic.js [hunk 2]
```diff
@@ -107,5 +108,11 @@ export default action$ =>
     ofType(actions.fetchWorker.TRIGGER),
     map(action => action.payload),
     throttleTime(1000),
-    switchMap(params => fetchWorker$(params)),
+    switchMap(params =>
+      fetchWorker$(params).pipe(
+        // avoid the fetching request for connector definitions is
+        // keep running after delete workspace triggered
```
Comment (konekoya) [inline ohara-manager/client/src/store/epics/worker/fetchWorkerEpic.js:114]: What do you think about "Stop this epic once the delete workspace action is triggered"

**Comment by konekoya** [comment]
Love this PR so much ❤️

**Comment by saivirtue** [comment]
retry manager-e2e

**Pull request merged**